sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationRefresh.java

```diff
@@ -107,16 +120,14 @@ private boolean refreshStores() {
      * @param store the {@code store} for which to composite watched key names
      * @return Refresh event was triggered. No other sources need to be checked.
      */
-    private boolean refresh(ConfigStore store) {
-        State state = StateHolder.getState(store.getEndpoint());
-
+    private boolean refresh(State state, String endpoint, Duration cacheTime) {
```

Comment:
please also update the java doc about the two newly added params. And what's this cacheTime used for?